app/src/main/java/org/wikipedia/suggestededits/SuggestedEditsCardsItemFragment.kt

```diff
@@ -28,38 +27,41 @@ import org.wikipedia.util.log.L
 import org.wikipedia.views.ImageZoomHelper
 
 class SuggestedEditsCardsItemFragment : SuggestedEditsItemFragment() {
+    private var _binding: FragmentSuggestedEditsCardsItemBinding? = null
+    private val binding get() = _binding!!
     var sourceSummaryForEdit: PageSummaryForEdit? = null
     var targetSummaryForEdit: PageSummaryForEdit? = null
     var addedContribution: String = ""
         internal set
 
-    override fun onCreateView(inflater: LayoutInflater, container: ViewGroup?, savedInstanceState: Bundle?): View? {
+    override fun onCreateView(inflater: LayoutInflater, container: ViewGroup?, savedInstanceState: Bundle?): View {
         super.onCreateView(inflater, container, savedInstanceState)
-        return inflater.inflate(R.layout.fragment_suggested_edits_cards_item, container, false)
+        _binding = FragmentSuggestedEditsCardsItemBinding.inflate(LayoutInflater.from(context), container, false)
```

Comment:
Can we just use the `inflater` parameter that is given to us by this function?

Reply:
Yes and done.